build.assets/build-package.sh

```diff
@@ -228,11 +228,11 @@ if [[ "${PACKAGE_TYPE}" == "pkg" ]]; then
     NOTARIZE_PKG="true"
     # handle mac client-only builds
     if [[ "${BUILD_MODE}" == "tsh" ]]; then
-        FILE_LIST="${TAR_PATH}/tsh"
+        FILE_LIST="${TAR_PATH}/tsh ${TAR_PATH}/tbot"
```

Comment:
`tbot` should probably only be included in the full tarball rather than the `tsh`-only package. Many customers only want `tsh` for providing to their end users whereas `tbot` is more of a daemon IIRC?

Reply:
Ah, fair point, I'll remove it.